### PR TITLE
Fix alert wait queue

### PR DESCRIPTION
**Pull request by ViniciusCampanha**
- Alert wait queue
- Alert PDF report
- Minor fixes

### CODE REVIEW

File: src/terrama2/services/alert/impl/DocumentPDF.hpp [hunk 1]
```diff
@@ -75,26 +91,18 @@ namespace terrama2
...
-                QString errMsg = QObject::tr("Couldn't create PDF document: Informed directorie doesn't exist!");
-                TERRAMA2_LOG_ERROR() << errMsg;
-                throw NotifierException() << ErrorDescription(errMsg);
-              }
 
-              std::string path = dir.absolutePath().toStdString() + "/"
-                                 + terrama2::core::simplifyString(report->name())
-                                 + ".pdf";
+              std::string filePath = dir.absolutePath().toStdString() + "/"
+                                     + terrama2::core::simplifyString(report->name())
```
Comment (janosimas) [inline src/terrama2/services/alert/impl/DocumentPDF.hpp]: I think it's best to use the "document_save_path" as complete path to the file, not just folder.

**Comment by ViniciusCampanha** [comment]
Jenkins rebuild

File: src/terrama2/services/alert/core/Report.cpp [hunk 2]
```diff
@@ -407,3 +407,17 @@ terrama2::core::DataSeriesType terrama2::services::alert::core::Report::dataSeri
 {
   return alertDataSeries_->semantics.dataSeriesType;
 }
+
+std::string terrama2::services::alert::core::Report::documentSavePath() const
+{
+  try
+  {
+    return alert_->reportMetadata.at("document_save_path");
```
Comment (janosimas) [inline src/terrama2/services/alert/core/Report.cpp]: use a const string instead of hard coded value.

**Comment by ViniciusCampanha** [comment]
Jenkins rebuild

**Comment by ViniciusCampanha** [comment]
Jenkins rebuild

**Pull request merged**